src/zenml/materializers/pandas_materializer.py

```diff
@@ -54,17 +54,23 @@ def handle_input(self, data_type: Type[Any]) -> pd.DataFrame:
         # Cleanup and return
         fileio.rmtree(temp_dir)
 
+        if data_type == pd.Series:
+            df = df[df.columns[0]]
+
         return df
 
-    def handle_return(self, df: pd.DataFrame) -> None:
-        """Writes a pandas dataframe to the specified filename.
+    def handle_return(self, df: Union[pd.DataFrame, pd.Series]) -> None:
+        """Writes a pandas dataframe or series to the specified filename.
 
         Args:
-            df: The pandas dataframe to write.
+            df: The pandas dataframe or series to write.
         """
         super().handle_return(df)
         filepath = os.path.join(self.artifact.uri, DEFAULT_FILENAME)
 
+        if type(df) == pd.Series:
+            df = df.to_frame(name="series")
```

Comment:
Is there any significance to the string "series"?

Reply:
No, it just needs to be a string for the `to_parquet()` to work. I can move it into a const field like how `DEFAULT_FILENAME` is done if you prefer.

Reply:
I ran into an issue where a series made the col name a datetimeindex which threw an error when the materializer called `.to_parquet()` setting the name to any valid string resolves this issue. Since we do not access the column by this name I didn't make it a variable.